tcms/testplans/static/testplans/js/search.js

```diff
@@ -189,9 +209,44 @@ export function pageTestplansSearchReadyHandler () {
         return false // so we don't actually send the form
     })
 
+    $('#btn_clone_bulk').click(function () {
+        const selectedTestPlans = getSelectedTestPlans()
+        if (selectedTestPlans.length === 0) {
+            alert('No test plans selected for cloning.')
```

Comment:
This string must be translatable. See how it's done for other strings on dynamic pages (`.data('trans-` selectors).